windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,87 @@
+
+On Windows, this folder is `C:\Certbot`. This can be changed using the `--config-dir` flag on the Certbot CLI.
+
+Within this folder you will find the following folders:
+* `C:\Certbot\live`: contains current versions of issued certificates and keys
+* `C:\Certbot\archive`: contains all current and previous versions of issued certificates and keys
+* `C:\Certificate\logs`: contains Certbot activity logs
+
+Doing backups of `C:\Certbot` folder is all that's required to save your Certbot installation. Please note that the `archive` folder is sensitive because it contains cryptographic private keys: this folder should not be disclosed to unauthorized parties.
+
+## Certificate renewals
+
+Renewals are supported by Certbot for Windows and are even already configured. Indeed the Certbot installer created a scheduled task during the installation to execute `certbot renew` twice a day.
+
+All certificates created with Certbot for Windows are then automatically renewed out-of-the-box, provided that the required environment to renew is available (e.g. credentials, running web server ...). Indeed Certbot remembers the method that you used to prove your control over the domain name at the time you originally created each certificate. This method is also used for automated renewals. Therefore, if your environment has changed since you first obtained your certificate (for example, if TCP port 80 is no longer free), you might need to change the associated authentication method in order for automated renewals to continue working.
+
+## A note on manual hooks
+
+As explained by the [manual plugin documentation](https://certbot.eff.org/docs/using.html#manual), you may need to set up authentication and cleanup hooks to make the `manual` plugin work as expected, in particular with `DNS-01` challenges.
+
+Similarly to Linux, Certbot for Windows supports any file executable for this platform, if it is available in the `PATH`. It can be `.bat`, `.ps1` or even executable binaries like `.exe`.
+
+## A note on files security
+
+Assets generated by Certbot, in particular keys, are critical in terms of security and should never be exposed to unauthorized accesses. Certbot for Windows leverages DACLs implemented by the NTFS filesystem to enforce this control: keys and sensitive materials are readable by default only by the account that runs Certbot, the `Administrators` group and `SYSTEM`.
+
+## Updating Certbot
+
+When a new version of Certbot release is available, please download the new installer for this new version as described in the _Installation_ section, and run it. The installer will update your existing installation of Certbot.
+
+## Getting help, reporting issues, asking for new features
+
+If you encounter some difficulties to install or use Certbot for Windows, there is a dedicated area in the Let's Encrypt Community Forum that can be reached at _[URL to dedicated category on Certbot (beta) for Windows in the Let's Encrypt Community Forum to be defined]_.
```

Comment:
Did we get approval to create our own dedicated category on the forum? It seems like a little much to me and I'm not sure Let's Encrypt would really like us hijacking their forum in this way.

Reply:
It is just a proposition, but nothing formal. If it is possible it is great, I would also go with a tag. It is to organize better the support. What are the acceptable options here?

Reply:
How about we just point people to the "Help" category of the forum and if we're getting a lot of threads, we can ask about making a dedicated category and edit the post with the new instructions?

Reply:
I did that.